pkgs/by-name/wo/worldpainter/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  jre,
+  gnused,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "worldpainter";
+  version = "2.22.1";
```

```suggestion
  version = "2.23.2";
```
New version released by now